backend/alembic/versions/c18cdf4b497e_add_standard_answer_tables.py

```diff
@@ -0,0 +1,68 @@
+"""Add standard_answer tables
+
+Revision ID: c18cdf4b497e
+Revises: b85f02ec1308
+Create Date: 2024-06-06 15:15:02.000648
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "c18cdf4b497e"
```

Comment:
I haven't verified but I suspect Pablo has some PRs which may also have migrations, depending on when this goes in, let's be sure it matches the version on main

Reply:
conflicts 😰 😱